src/main/java/org/ibp/api/security/WorkbenchUserDetailsService.java

```diff
@@ -46,25 +49,23 @@ public UserDetails loadUserByUsername(String username) throws UsernameNotFoundEx
 				final WorkbenchUser workbenchUser = matchingUsers.get(0);
 				// FIXME Populate flags for accountNonExpired, credentialsNonExpired, accountNonLocked properly, all true for now.
 				return new org.springframework.security.core.userdetails.User(workbenchUser.getName(), workbenchUser.getPassword(),
-						this.getRolesAsAuthorities(workbenchUser));
+						this.getAuthorities(workbenchUser));
 			}
 			throw new UsernameNotFoundException("Invalid username/password.");
 		} catch (final MiddlewareQueryException e) {
 			throw new AuthenticationServiceException("Data access error while authenticaing user against Workbench.", e);
 		}
 	}
 
-	private Collection<? extends GrantedAuthority> getRolesAsAuthorities(final WorkbenchUser workbenchUser) {
-		final List<GrantedAuthority> authorities = new ArrayList<GrantedAuthority>();
-		if (workbenchUser != null) {
-			final List<UserRole> userRoles = workbenchUser.getRoles();
-			if (userRoles != null && !userRoles.isEmpty()) {
-				for (final UserRole role : userRoles) {
-					authorities.add(new SimpleGrantedAuthority(role.getRole().getCapitalizedRole()));
-				}
-			}
-		}
+	private Collection<? extends GrantedAuthority> getAuthorities(final WorkbenchUser workbenchUser) {
```

Comment:
Question: Why are we maintaining two `WorkbenchUserDetailsService `implementations, one in Commons and one in BMSAPI? The one in Commons was able to pass crop and program to the permissionService.